index.css

```diff
@@ -0,0 +1,142 @@
+/* colors
+--clr-navbar-bg:22 1% 24% 1;
+--clr-base-bg:22 6% 10% 1;
+--clr-heading:22 100% 48%;
+--clr-text:0, 0%, 100%, 1;
+*/
```

Comment:
Great job @shuja-shah ✔️.

- [ ] It is not a good practice to leave commented code within your project. Please try to remove all the commented code from your style sheet.